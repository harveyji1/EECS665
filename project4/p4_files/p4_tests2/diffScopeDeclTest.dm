 b : int;
 v : () int{ 
    b : int;
}