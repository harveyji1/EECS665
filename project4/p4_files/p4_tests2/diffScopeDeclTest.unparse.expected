b{int} : int;
v{()->int} : () int  {
    b{int} : int;
}
